.github/workflows/clear_self_hosted_persistent_caches.yaml

```diff
@@ -26,6 +26,25 @@ jobs:
       run: du -sh ~/.pex || true; rm -rf ~/.pex || true
     - name: df after
       run: df -h
+  clean_macos13_x86_64:
```

Comment:
What's the purpose of deleting everything associated? 
nce, .cache/pants, and I guess rust/pex make sense - but why everything else?

Reply:
That's what our current GHA generator script does for self-hosted runners. I did not do a deep dive into it.